src/Sources/Sedimentation.jl

```diff
@@ -5,6 +5,7 @@
 """
 module Sedimentation
 
+using Cloudy
```

Comment:
```suggestion
import ..: rflatten
```
This is not a safe way to do code loading, as `using Cloudy` could (potentially) pick up a local (and different) version of `Cloudy`. We can use `..` notation to load/use things from the parent module.

The syntax I've given may not be exactly correct-- I have a hard time remembering grabbing types/functions from the parent scope. I always seem to remember neighboring scope, though: if `rflatten exists in `src/Utils.jl` in a `Utils` module, then `import ..Utils: rflatten` will work.